### PR TITLE
Update ticket pattern defaults in schema and jira.py

### PR DESCRIPTION
**Pull request by MatthijsBon**
Sorry, I hadn't configured my git hooks yet 😕 

The Marketplace Growth team has a project with prefix "MG", which is two characters, meaning our ticket weren't recognized by mpyl. This fixes that.

### CODE REVIEW

File: src/mpyl/reporting/targets/jira.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def from_config(config: dict):
         if not jira_config:
             raise KeyError('jira section needs to be defined in mpyl_config.yml')
         return JiraConfig(site=jira_config['site'], user_name=jira_config['userName'], password=jira_config['password'],
-                          ticket_pattern=re.compile(jira_config.get('ticketPattern', '[A-Za-z]{3,}-\\d+')),
+                          ticket_pattern=re.compile(jira_config.get('ticketPattern', '[A-Za-z]{2,}-\\d+')),
```
Comment (SamTheisens) [inline src/mpyl/reporting/targets/jira.py:92]: This is just a fallback for when no pattern is configured. For the monorepo the pattern is configured here https://github.com/Vandebron/Vandebron/blob/c2e87c320916d74931639b1166f52289fd1a256f/mpyl_config.yml#L20.

**Review by SamTheisens — APPROVED** [review]
Looks good, but you'll need to update the monorepo too.

File: src/mpyl/reporting/targets/jira.py (hunk 1 above)
Comment (MatthijsBon) [reply src/mpyl/reporting/targets/jira.py:92]: Did: https://github.com/Vandebron/Vandebron/pull/10495 :)_

Comment (koeves) [inline src/mpyl/reporting/targets/jira.py:92]: Can we add a unit test for this?

Comment (SamTheisens) [reply src/mpyl/reporting/targets/jira.py:92]: The tests are there, just need to be linked up with the default config:
https://github.com/Vandebron/mpyl/blob/97a5d872414aae2d59c51db7e6a609c9acd8c5b9/tests/reporting/targets/test_jira.py#L45-L54

**Review by SamTheisens — APPROVED** [review]

**Pull request merged**